### PR TITLE
Support colord

### LINKED ISSUE #44: Colord can't load an ICC profile.
I've been having problems matching prints to my monitor, even though everything is calibrated. 

It seems that colord, though built into the flatpak, can't actually load a profile:

```
flatpak run --command=darktable-cmstest org.darktable.Darktable 
darktable-cmstest version 2.6.2
this executable was built with colord support enabled
darktable itself was built with colord support enabled

primary CRTC is at CRTC 0
CRTC for screen 0 CRTC 1 has no mode or no output, skipping
CRTC for screen 0 CRTC 2 has no mode or no output, skipping
error connecting to colord

eDP-1   the X atom and colord returned different profiles
        X atom: _ICC_PROFILE (1152 bytes)
                description: Galago Pro
        colord: "(none)"
                description: (file not found)

Better check your system setup
 - some monitors reported different profiles
You may experience inconsistent color rendition between color managed applications
```

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Fixes #44

### CODE REVIEW

File: org.darktable.Darktable.json [hunk 1]
```diff
@@ -13,10 +13,8 @@
         "--socket=x11",
         "--filesystem=host",
         "--talk-name=org.freedesktop.secrets",
-        /* Needed for dconf to work */
-        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
-        "--filesystem=xdg-run/gvfs",
-        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
```
Comment (hfiguiere) [inline org.darktable.Darktable.json]: It was my understanding that this was for when runtime 3.34 is used. darktable still has moved to 3.34 for a few reasons.

**Comment by paperdigits** [comment]
@Hofer-Julian  if I run darktable-cmtest inside the flatpak sandbox, will it report back the monitor profile set by the system's colord?

File: org.darktable.Darktable.json (hunk 1 above)
Comment (Hofer-Julian) [reply org.darktable.Darktable.json]: Nice catch!
I'll revert this change.

**Comment by Hofer-Julian** [comment]
> @Hofer-Julian if I run darktable-cmtest inside the flatpak sandbox, will it report back the monitor profile set by the system's colord?

Yes, you can try
 ```
 flatpak install --user https://dl.flathub.org/build-repo/14083/org.darktable.Darktable.flatpakref
 ```
to check if it also works on your system.

**Comment by Hofer-Julian** [comment]
I removed the dconf permissions again, as you use 3.34 now

**Comment by hfiguiere** [comment]
Yes we do use 3.34 runtime, but it is also Darktable 3.0 now. You actually need to rebase the pull request on the current master.

**Comment by hfiguiere** [comment]
For the record the current darktable package show this:
```
flatpak run --command=darktable-cmstest org.darktable.Darktable
darktable-cmstest version 3.0.0
this executable was built with colord support enabled
darktable itself was built with colord support enabled

couldn't locate primary CRTC!
error connecting to colord

XWAYLAND0	the X atom and colord returned the same profile
	X atom:	_ICC_PROFILE (0 bytes)
		description: (none)
	colord:	"(none)"
		description: (file not found)

Better check your system setup
 - some monitors lacked a profile
You may experience inconsistent color rendition between color managed applications
```

(I don't have a profile set on this machine, but it show the colord error)

**Comment by Hofer-Julian** [comment]
> Yes we do use 3.34 runtime, but it is also Darktable 3.0 now. You actually need to rebase the pull request on the current master.

@hfiguiere Done.

**Comment by Hofer-Julian** [comment]
My output:
```
flatpak run --command=sh org.darktable.Darktable 
[📦 org.darktable.Darktable ~]$ darktable-cmstest 
darktable-cmstest version 3.0.0
this executable was built with colord support enabled
darktable itself was built with colord support enabled

primary CRTC is at CRTC 0
CRTC for screen 0 CRTC 1 has no mode or no output, skipping
CRTC for screen 0 CRTC 2 has no mode or no output, skipping

HDMI-1	the X atom and colord returned the same profile
	X atom:	_ICC_PROFILE (1460 bytes)
		description: 23MB35
	colord:	"/home/julian/.local/share/icc/edid-0a7fad8ba3e1ea6263c93e2a07623246.icc"
		description: 23MB35

Your system seems to be correctly configured

```

File: org.darktable.Darktable.json [hunk 2]
```diff
@@ -13,12 +13,11 @@
...
```
Comment (hfiguiere) [inline org.darktable.Darktable.json]: But it is missing the migration as explained at:

https://blogs.gnome.org/mclasen/2019/07/12/settings-in-a-sandbox-world/

Comment (Hofer-Julian) [reply org.darktable.Darktable.json]: Does darktable even use dconf?
Using the dconf editor, I couldn't find anything from darktable.

Comment (Hofer-Julian) [reply org.darktable.Darktable.json]: To be safe, I can still add the line, if you want me to.

Comment (hfiguiere) [reply org.darktable.Darktable.json]: I believe it should.

Comment (Hofer-Julian) [reply org.darktable.Darktable.json]: Done.

**Comment by hfiguiere** [comment]
Thank you so much.

**Pull request merged**